src/feature/dirauth/process_descs.c

```diff
@@ -237,9 +296,9 @@ dirserv_router_get_status(const routerinfo_t *router, const char **msg,
   }
 
   /* Check for the more usual versions to reject a router first. */
-  const uint32_t r = dirserv_get_status_impl(d, router->nickname,
-                                             router->addr, router->or_port,
-                                             router->platform, msg, severity);
+  uint32_t r = dirserv_get_status_impl(d, router->nickname, router->addr,
```

Comment:
I don't understand why you removed this const.
If you're going to make unrelated changes, make them in a separate commit?